api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/ObservationValueTranslatorImplTest.java

```diff
@@ -48,6 +50,8 @@
 	
 	private static final String CONCEPT_VALUE_UUID = "12345-abcde-54321";
 	
+	private static final String LOCATION_UUID = "2321gh23-kj34h45-34jk3-34k34k";
```

Comment:
We probably should have a test at the `ObservationTranslatorImplTest` level anyway to verify that the Observation gets created properly.